src/Features/VisualBasic/Portable/Completion/CompletionProviders/CrefCompletionProvider.vb

```diff
@@ -10,6 +10,7 @@ Imports Microsoft.CodeAnalysis.Text
 Imports Microsoft.CodeAnalysis.VisualBasic.Extensions.ContextQuery
 Imports Microsoft.CodeAnalysis.VisualBasic.Syntax
 Imports System.Collections.Immutable
+Imports Microsoft.CodeAnalysis.ErrorReporting
```

Comment:
sort.

Reply:
This was added by Add Import refactoring. Sounds like a bug.

Reply:
THe imports were already sorted incorrectly (System.Collections.Immutable at the bottom).  Add import won't sort if things are unsorted.

Reply:
Done.